src/_js/main.mjs

```diff
@@ -59,6 +59,14 @@ if (location.search.includes('utm_source')) {
   history.replaceState({}, '', location.pathname);
 }
 
+// Helper function to dynamically insert scripts.
+const insertScript = (src) => {
+  const firstScript = document.scripts[0];
```

Comment:
Let’s hoist this one out of the function.

Reply:
Done https://github.com/v8/v8.dev/pull/254/commits/b3dadcae605cdf26e322cbd929b20c7f428e31be